lib/cove/command/builder.rb

```diff
@@ -7,6 +7,10 @@ def self.start_container(*containers)
         [:docker, "container", "start", *containers.flatten]
       end
 
+      def self.start_attached_container(container)
```

Comment:
```suggestion
      def self.start_container_and_attach(container)
```

Feels more descriptive for what is going on.